src/components/editor/faultTree/menu/FaultEventScenariosTable.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { useEffect, useState, FC } from "react";
+import { Box, Typography, useTheme } from "@mui/material";
+import { FaultEventScenario } from "../../../../models/faultEventScenario";
+import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Paper } from "@mui/material";
+import useStyles from "./FaultEventScenariosTable.styles";
+import { asArray } from "@utils/treeUtils";
```

Comment:
Although it is confusing that it belongs to treeUtils as it does not have anything to do with trees, but it is acceptable to me like that. I would rather add it to a new class called CollectionsUtils, ArrayUtils or just Utils.